sharks/src/share_ff.rs

```diff
@@ -225,11 +225,12 @@ mod tests {
     let iter =
       get_evaluator(vec![vec![fp_three(), fp_two(), fp_three() + fp_two()]]);
     let values: Vec<(Fp, Vec<Fp>)> = iter.take(2).map(|s| (s.x, s.y)).collect();
+    println!("values: {:?}", values[1].1[0].0);
```

Comment:
It's in a test, so this output isn't normally visible, but I think it's worth removing the scaffolding.